_episodes/10-collaborating.md

```diff
@@ -39,45 +40,59 @@ Next, the Collaborator needs to download a copy of the Owner's repository to her
 her `Desktop` folder, the Collaborator enters:
 
 ~~~
-$ git clone https://github.com/vlad/planets.git ~/Desktop/vlad-planets
+$ git clone https://github.com/ccline/cats-as-data.git ~/Desktop/ccline-cats-as-data
 ~~~
 {: .bash}
 
-Replace 'vlad' with the Owner's username.
+Replace 'ccline' with the Owner's username. Also note the username prefix on the directory name. This is done because
```

Comment:
Nit: backticks around `ccline`?

Reply:
You're really ticing me off. yes fine :)